### PR TITLE
Add enabling default instructions flag for mobile

### PR DESCRIPTION
**Pull request by tugsbayasgalan**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#57778 Add enabling default instructions flag for mobile**

Differential Revision: [D28268997](https://our.internmc.facebook.com/intern/diff/D28268997)

### CODE REVIEW

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28268997).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28268997).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28268997).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28268997).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28268997).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28268997).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28268997).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28268997).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28268997).

File: torch/csrc/jit/serialization/export_module.cpp [hunk 1]
```diff
@@ -69,7 +69,7 @@ std::pair<IValue, IValue> getFunctionTuple(
 
   Inline(*graph);
 
-  torch::jit::MobileCode code(graph, func.name());
+  torch::jit::MobileCode code(graph, func.name(), true);
```
Comment (gmagogsfm) [inline torch/csrc/jit/serialization/export_module.cpp:72]:
```suggestion
  torch::jit::MobileCode code(graph, func.name(), /*emit_default_input_instructions=*/true);
```

File: torch/csrc/jit/runtime/interpreter.h [hunk 2]
```diff
@@ -81,6 +81,7 @@ struct TORCH_API MobileCode : Code {
   explicit MobileCode(
       const std::shared_ptr<Graph>& graph,
       std::string function_name,
+      bool emit_default_input_instructions = false,
```
Comment (gmagogsfm) [inline torch/csrc/jit/runtime/interpreter.h:84]: Maybe the default value should be `True`. When we decide to enable FC for mobile, we can flip value used on line 72 of export_module.cpp to `False`. Eventually, this flag can be removed.

**Review by iseeyuan — APPROVED** [review]
LGTM. Thanks!

**Review by gmagogsfm — APPROVED** [review]

**Pull request closed without merging**